### PR TITLE
Update installation doc

### PR DESCRIPTION
**Pull request by pierrepo**
Try to make doc clearer.

On Mac OSX, the ``$PATH`` update might be:

    export PATH=~/Library/Python/X.Y/bin:$PATH

where ``X.Y`` is the Python version (``2.7`` for Python 2.7 and ``3.5`` for Python 3.5).

@HubLot can you have a look with @alexdb27 and update the PR accordingly?

### CODE REVIEW

**Comment by alexdb27** [comment]
by default is it the 2.7 ?? (on my mac it is)

**Comment by pierrepo** [comment]
@hublot I updated the PR. Does this sound better to you?

**Comment by HubLot** [comment]
Yep, it's good. 
I will add some modifications we discussed with Alex

**Comment by pierrepo** [comment]
OK. Go!

File: doc/source/installation.rst [hunk 1]
```diff
@@ -40,20 +40,42 @@ The most straightforward way is to use `pip`:
...
 
-    $ pip install pbxplore
+    $ pip install --user pbxplore
 
 
-PBxplore can also be installed for the current user only:
+.. note::
 
-.. code-block:: bash
+    The former command with install PBxplore in:
```
Comment (HubLot) [inline doc/source/installation.rst]: "with" ? :)

Comment (pierrepo) [inline doc/source/installation.rst]: ``will`` ?

Comment (HubLot) [inline doc/source/installation.rst]: smith ?

Comment (pierrepo) [inline doc/source/installation.rst]: smooth ?

Comment (pierrepo) [inline doc/source/installation.rst]: will ! definitely will !

Comment (HubLot) [inline doc/source/installation.rst]: deal !

**Comment by HubLot** [comment]
done!

**Pull request merged**